docs/source/tutorials/multiannotator.ipynb

```diff
@@ -0,0 +1,555 @@
+    "from sklearn.model_selection import cross_val_predict\n",
+    "\n",
+    "model = LogisticRegression()\n",
+    "\n",
+    "num_crossval_folds = 5  \n",
+    "cv_pred_probs = cross_val_predict(\n",
+    "    estimator=model, X=X, y=majority_vote_label, cv=num_crossval_folds, method=\"predict_proba\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4eab5188",
+   "metadata": {},
+   "source": [
+    "## 4. Use cleanlab to get better consensus labels and other statistics"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d392ce5",
+   "metadata": {},
+   "source": [
+    "Using the annotators' labels and the out-of-sample predicted probabilites from the model, cleanlab can help us obtain improved consensus labels for our data.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6ca92617",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "multiannotator_dict = get_label_quality_multiannotator(multiannotator_labels, cv_pred_probs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "98042e7f",
+   "metadata": {},
+   "source": [
+    "Here, we use the `multiannotator.get_label_quality_multiannotator()` function which returns a dictionary containing three items:\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "76d7c0e2",
+   "metadata": {},
+   "source": [
+    "- `label_quality_multiannotator` which gives us the improved consensus labels using information from each of the annotators and the model. The DataFrame also contains information about the number of annotations, annotator agreement and consensus quality score for each example.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bf945113",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "multiannotator_dict[\"label_quality_multiannotator\"].head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "984d65c4",
+   "metadata": {},
+   "source": [
+    "- `detailed_label_quality` which returns the label quality score for each label given by every annotator"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "14251ee0",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "multiannotator_dict[\"detailed_label_quality\"].head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "db02e63d",
+   "metadata": {},
+   "source": [
+    "- `annotator_stats` which gives us the annotator quality score for each annotator, alongisde other information such as the number of examples each annotator labeled, their agreement with the consensus label and the class they perform the worst at. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "efe16638",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "multiannotator_dict[\"annotator_stats\"].head(10)"
+   ]
```

Comment:
what do you think about renaming the column "num_labeled" -> "num_examples_labeled" ?

Reply:
maybe let's reorder the columns more thematically so the column order is:

ID, annotator_quality, agreement_with_consensus, num_labeled, worst_class

What do you think?

Reply:
Can we make the annotators have string IDs in the original dataset, and those IDs should be shown here as well in this DF.

If the annotators have IDs, the IDs should also be the suffix of each column name in:
`multiannotator_dict["detailed_label_quality"]`

Reply:
I agree that `num_examples_labeled` would be a clearer name.
I'm thinking if we should order `num_examples_labeled` as the last column (it would be consistent with `num_annotations` being the last column in the `label_quality` df)

Reply:
Sounds good!

Reply:
Are you proposing that `ID` is a column of its own in the `annotator_stats` df? 
Currently the annotator ID is used as the index (right now they have int IDs so it is not obvious), and the column names of `detailed_label_quality` is `quality_annotator_{annotatorID}`. Is that alright or should something be changed in terms of displaying IDs?
Will edit the tutorial annotator IDs to be strings instead of ints.

Reply:
No sorry for confusion, let's just keep ID as string index when it exists!

I just wanted to make sure ID will show up somewhere if it is string